service/src/main/java/com/fit_goal/service/AuditService.java

```diff
@@ -0,0 +1,44 @@
+package com.fit_goal.service;
+
+import com.fit_goal.Audit;
+import com.fit_goal.AuditDao;
+import com.fit_goal.domain.AuditDto;
+
+import javax.inject.Inject;
+import java.util.List;
+import java.util.Optional;
+
+public class AuditService implements Audit {
```

Comment:
Please stick to the following pattern for visibility: 
Interface for service -> AuditService
Implementation -> AuditServiceImpl

Reply:
refactored